kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/JdbcColumn.java

```diff
@@ -240,6 +241,10 @@ static String getColumnTypeName(TTypeId type) throws SQLException {
         return "struct";
       case NULL_TYPE:
         return "void";
+      case UNION_TYPE:
+        return "uniontype";
```

Comment:
UNION_TYPE was missed here before.
```
  UNION_TYPE("UNIONTYPE",
      java.sql.Types.OTHER,
      TTypeId.UNION_TYPE,
      true, false),
```

Reply:
the original copy is based on 2.3 / 3.1, if things happen after 3.1, we'd better leave a Hive JIRA ticket in the comments

Reply:
It has been 4 years since last commit for Type.java.
<img width="1684" alt="image" src="https://github.com/apache/kyuubi/assets/6757692/6c2fc8dc-1f41-44e3-a7f6-dd81d45558e6">

Reply:
https://github.com/apache/hive/blob/master/serde/src/java/org/apache/hadoop/hive/serde2/thrift/Type.java

Reply:
Missed here: https://github.com/apache/kyuubi/pull/2878/files#r1538554887

Reply:
should I raise another pr to address it? @pan3793

Reply:
let's fix it first, independently